Fix `test_agent_execution`

Since merging #8750, we're getting intermittent CI test failures of `test_agent_execution`:

[https://github.com/Significant-Gravitas/AutoGPT/actions/runs/12400120333/job/34616544224](https://github.com/Significant-Gravitas/AutoGPT/actions/runs/12400120333/job/34616544224)

`FAILED test/executor/test_manager.py::test_agent_execution - AssertionError: assert 'e2835f4c-b713-45c0-91fd-28217d5f468c' in ['07c930f8-9066-4444-960d-0ffb4d1a2ece', '085c2275-ae1a-4b7d-b442-5eb2d3579f8c']`<br>`+ where 'e2835f4c-b713-45c0-91fd-28217d5f468c' = ExecutionResult(graph_id='ac4dafae-85af-4d78-a2cf-65ce103dbd53', graph_version=1, graph_exec_id='bd69bdba-012b-4538-80ec-6f171cc3f82d', node_exec_id='98e923d9-f050-4b5d-bd8e-46ed8b3f4ab3', node_id='e2835f4c-b713-45c0-91fd-28217d5f468c', block_id='c0a8e994-ebf1-4a9c-a4d8-89d09c86741b', status=<AgentExecutionStatus.COMPLETED: 'COMPLETED'>, input_data={'name': 'input_2', 'value': 'World'}, output_data={'result': ['World']}, add_time=datetime.datetime(2024, 12, 18, 19, 43, 9, 67000, tzinfo=TzInfo(UTC)), queue_time=datetime.datetime(2024, 12, 18, 19, 43, 9, 87000, tzinfo=TzInfo(UTC)), start_time=datetime.datetime(2024, 12, 18, 19, 43, 25, 305000, tzinfo=TzInfo(UTC)), end_time=datetime.datetime(2024, 12, 18, 19, 43, 25, 418000, tzinfo=TzInfo(UTC))).node_id`

--- a/autogpt_platform/backend/backend/data/graph.py
+++ b/autogpt_platform/backend/backend/data/graph.py
@@ -629,23 +629,18 @@ async def __create_graph(tx, graph: Graph, user_id: str):
             "isTemplate": graph.is_template,
             "isActive": graph.is_active,
             "userId": user_id,
+            "AgentNodes": {
+                "create": [
+                    {
+                        "id": node.id,
+                        "agentBlockId": node.block_id,
+                        "constantInput": json.dumps(node.input_default),
+                        "metadata": json.dumps(node.metadata),
+                    }
+                    for node in graph.nodes
+                ]
+            },
         }
-    )
-
-    await asyncio.gather(
-        *[
-            AgentNode.prisma(tx).create(
-                {
-                    "id": node.id,
-                    "agentBlockId": node.block_id,
-                    "agentGraphId": graph.id,
-                    "agentGraphVersion": graph.version,
-                    "constantInput": json.dumps(node.input_default),
-                    "metadata": json.dumps(node.metadata),
-                }
-            )
-            for node in graph.nodes
-        ]
     )
 
     await asyncio.gather(

